deploy/capabilities-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "displayFields": [
+    {
+      "field": "status",
+      "display": "Status"
+    },
+    {
+      "field": "submission",
+      "display": "Submitted"
+    },
+    {
+      "field": "end",
+      "display": "End Time"
+    },
+    {
+      "field": "labels.cromwell-workflow-id",
```

Comment:
Should this file (and `api-config.json`) be within the cromwell folder since its specific to that deployment?

Reply:
Good catch, will move. Besides, as I said above, do you think it would be a good idea to make changes to dsub shim so it can also accept JSON capabilities configs?